freqtrade/strategy/interface.py

```diff
@@ -127,6 +133,19 @@ def populate_sell_trend(self, dataframe: DataFrame, metadata: dict) -> DataFrame
         :return: DataFrame with sell column
         """
 
+    def additional_pairs(self) -> List[Tuple[str, str]]:
```

Comment:
I am not a 100% comfortable with the naming àdditional_pairs`, I would name it informational_pairs, nontradable_pairs or something else. but this is just me honestly.

Reply:
Informative_pairs sounds better - i always said, I'm not good at naming things...